openstack/image/api.go

```diff
@@ -253,6 +267,8 @@ func (h APIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 // on return values all the time.
 func errorResponse(err error) APIResponse {
 	switch err {
+	case ErrNoImage:
+		return APIResponse{http.StatusNotFound, nil}
 	default:
```

Comment:
@obedmr ErrImageSaving is returned from UploadImage - which is a PUT method. This error is encountered if the state of the image is already in "Saving" when someone tries to do another upload. Correct?